sig-wg-lifecycle.md

```diff
@@ -19,13 +19,26 @@ Out of scope for this document: [subproject] creation.
 - [ ] Read [wg-governance.md]
 - [ ] Ensure all WG Organizers, and other leadership roles are [community members]
 - [ ] Send email to [kubernetes-dev@googlegroups.com] titled "WG-Creation-Request: WG Foo" with some of the questions answered from wg-goverance.md and wait for community discourse; ask for SIG sponsorship
-- [ ] Do the first checklist item in the #GitHub section below and add a row to the WG section:
+- [ ] Do the first checklist item in the [GitHub] section below and add a row to the WG section:
   - [ ] Label with committee/steering and wait for a simple majority
   - [ ] Also add sponsoring SIG Chair/Tech Leads as approvers; you'll get this from the community email above
   - [ ] Place a `/hold` on it until the members that need to review have; a contributor experience member will do this for you if they don't see one already
 - [ ] Send an email to the stakeholder SIG mailing lists and steering committee with the sigs.yaml pull request
 
-### GitHub:
+### Prerequisites for a UG
+- [ ] Read [ug-governance.md]
+- [ ] Require to have 2 Chairs to start a User Group.
+- [ ] Require to have at least 2 members other than chairs to be a part of the User Group.
+- [ ] Send an email to kubernetes-dev@googlegroups.com and steering@kubernetes.io titled "UG-Creation-Request: UG Foo" answering the following questions and wait for community discourse:
+  - [ ] What topics are in scope for this user group?
+  - [ ] What is the meeting cadence?
+  - [ ] Who will chair the group, and ensure it continues to meet the requirements?
+- [ ] Do the first checklist item in the [GitHub] section below and add a row to the UG section:
+  - [ ] Label with committee/steering and place a `/hold` on it.
+- [ ] Send an email to the steering committee with the [sigs.yaml] pull request.
```

Comment:
I don't think steering needs to vote on UG formation, if the sponsoring SIG votes that should be sufficient.

Reply:
addressed on 4/24 steering